cmd/oceantv/main.go

```diff
@@ -223,16 +223,25 @@ func setup(ctx context.Context) {
 
 // tvRecipients looks up the email addresses and notification period
 // for the given site,
-// TODO: Use the notification kind for improved granularity.
 func tvRecipients(skey int64, kind notify.Kind) ([]string, time.Duration, error) {
 	ctx := context.Background()
 	site, err := model.GetSite(ctx, settingsStore, skey)
 	if err != nil {
 		return nil, 0, fmt.Errorf("error getting site: %w", err)
 	}
+	if site.OpsEmail == "" {
+		log.Printf("OpsEmail not defined for site %s", site.Name)
+	}
 	recipients := []string{site.OpsEmail}
```

Comment:
site.OpsEmail will still be added if it is empty here. Recipients should be an empty slice if OpsEmail is empty.

Reply:
Empty strings are ignored by the notifier.